guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -16,24 +16,38 @@
     "<ul class=\"toc-item\">\n",
```

Comment:
<blockquote>&nbsp;that can be used for inferencing on unseen data. Saved models can also be loaded back using the&nbsp;<code>load()</code>&nbsp;method. The&nbsp;<code>load()</code>&nbsp;method takes the path to the emd file as a required argument.</blockquote><p>Should load be replaced with from_model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Updated the section. `from_model` is used instead of `load`  method to load back a saved model.